spec/features/exhibits/translation_editing_spec.rb

```diff
@@ -71,6 +71,17 @@
   describe 'Search field labels' do
     before { visit spotlight.edit_exhibit_translations_path(exhibit, language: 'fr') }
 
+    it 'redirects to the same form tab' do
+      click_link 'Search field labels'
+      within('#search_fields', visible: true) do
+        fill_in 'Everything', with: 'Tout'
+        click_button 'Save changes'
+      end
+
+      expect(page).to have_css '.nav-pills li.active', text: 'French'
+      expect(page).to have_css '.nav-tabs li.active', text: 'Search field labels'
```

Comment:
@mejackreed what do you think about this (a separate test for redirects) vs. throwing in three one-liner expectations? e.g. adding `expect(page).to have_css '.nav-tabs li.active', text: 'Browse categories'` to the existing tests for: 
- Field-based search fields
- facet fields
- sort fields

Reply:
I think thats a great idea. One thing that might be nice to keep is the no javascript on the added tests.

Reply:
Neat. I've moved the spec for the Browse categories tab out of the JS tests.